go/test/endtoend/tabletgateway/vtgate_test.go

```diff
@@ -68,6 +68,40 @@ func TestVtgateReplicationStatusCheck(t *testing.T) {
 	expectNumRows := 2
 	numRows := len(qr.Rows)
 	assert.Equal(t, expectNumRows, numRows, fmt.Sprintf("wrong number of results from show vitess_replication_status. Expected %d, got %d", expectNumRows, numRows))
+
+	// Stop any VTOrc(s) so that it doesn't immediately repair/restart replication.
```

Comment:
you can just disable and enable recoveries on the VTOrc, don't have to stop and start it.

Reply:
`utils.DisableGlobalRecoveries(t, clusterInfo.ClusterInstance.VTOrcProcesses[0])` is used to disable recoveries in vtorc tests

Reply:
Done. Thanks!